### PR TITLE
Convert stored procedures to SQLX and globally deploy them in bqutil project

### PR DESCRIPTION
**Pull request by danieldeleo**
- Add stored procedures as dataform sqlx assets that deploy globally
- Fix dataform unit tests for dataform versions >=2.8.2
- Make JS_BUCKET paths unique to each commit

### CODE REVIEW

File: udfs/community/README.md [hunk 1]
```diff
@@ -167,11 +167,11 @@ SELECT bqutil.fn.int(1.684)
...
-* [sure_cond](#sure_condvalue-string-cond-bool)
-* [sure_like](#sure_likevalue-string-like_pattern-string)
-* [sure_nonnull](#sure_nonnullvalue-any-type)
-* [sure_range](#sure_rangevalue-any-type)
-* [sure_values](#sure_valuesvalue-any-type-acceptable_value_array-any-type)
+* [sure_cond](#sure_cond)
+* [sure_like](#sure_like)
+* [sure_nonnull](#sure_nonnull)
+* [sure_range](#sure_range)
+* [sure_values](#sure_values)
```
Comment (afleisc) [inline udfs/community/README.md:174]: I think we want to keep the links as they were to function correctly

Comment (danieldeleo) [reply udfs/community/README.md:174]: https://github.com/GoogleCloudPlatform/bigquery-utils/pull/439

Comment (danieldeleo) [reply udfs/community/README.md:174]: ah I didn't realize my PR reverted the PR

File: stored_procedures/definitions/get_next_ids.sqlx
Comment (afleisc) [inline stored_procedures/definitions/get_next_ids.sqlx:1]: Nit: missing copyright

**Review by afleisc — APPROVED** [review]

**Pull request merged**